airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/util/Jsons.kt

```diff
@@ -58,32 +55,4 @@ object Jsons : ObjectMapper() {
     }
 
     fun booleanNode(boolean: Boolean): BooleanNode = nodeFactory.booleanNode(boolean)
-
-    fun <T> `object`(jsonNode: JsonNode?, klass: Class<T>?): T? {
-        return convertValue(jsonNode, klass)
-    }
-
-    fun <T> serialize(`object`: T): String {
-        try {
-            return writeValueAsString(`object`)
-        } catch (e: JsonProcessingException) {
-            throw RuntimeException(e)
-        }
-    }
-
-    fun <T> deserialize(jsonString: String?, klass: Class<T>?): T {
-        try {
-            return readValue(jsonString, klass)
-        } catch (e: IOException) {
-            throw RuntimeException(e)
-        }
-    }
-
-    fun deserialize(jsonString: String?): JsonNode {
-        try {
-            return readTree(jsonString)
-        } catch (e: IOException) {
-            throw RuntimeException(e)
-        }
-    }
```

Comment:
These aren't necessary, let's stick to using the proper jackson methods instead.